runtime/src/main/java/com/flipkart/flux/dao/AuditDAOImpl.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2012-2016, the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.flipkart.flux.dao;
+
+import com.flipkart.flux.dao.iface.AuditDAO;
+import com.flipkart.flux.domain.AuditRecord;
+import com.flipkart.flux.util.HibernateUtil;
+import org.hibernate.Criteria;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.criterion.Restrictions;
+
+import java.util.List;
+
+/**
+ * @author shyam.akirala
+ */
+public class AuditDAOImpl extends AbstractDAO<AuditRecord> implements AuditDAO {
+
+    @Override
+    public AuditRecord findById(Long id) {
```

Comment:
You can take the type of the class in the constructor of the AbstractDAO and thus you will not need to unnecessarily implement these methods that do nothing but call the super() with the class reference